README: add details about the two default bottlerocket volumes (#1588)
bottlerocket-os/bottlerocket: README.md

The server and client are the user-facing components of the API system, but there are a number of other components that work together to make sure your settings are applied, and that they survive upgrades of Bottlerocket.

For more details, see the [API system documentation](sources/api/).

### Default Volumes

Bottlerocket operates with two default storage volumes. 

> * The root device, `/dev/xvda`, holds the active and passive [partition sets](#updates).

Reviewer: ```suggestion
* The root device, `/dev/xvda`, holds the active and passive [partition sets](#updates-1).
```
Sorry, I should have tested this link suggestion.  There are multiple sections named "updates" and I intended to point to the one shortly above this section that discusses partition sets.

The server and client are the user-facing components of the API system, but there are a number of other components that work together to make sure your settings are applied, and that they survive upgrades of Bottlerocket.

For more details, see the [API system documentation](sources/api/).

### Default Volumes

Bottlerocket operates with two default storage volumes. 

> * The root device, `/dev/xvda`, is immutable and is mostly used for holding the OS images with its active and passive partition sets.

Reviewer: ```suggestion
* The root device, `/dev/xvda`, holds the active and passive [partition sets](#updates).
```

>   It also contains the bootloader, the dm-verity hash tree for verifying the root filesystem image, and the data store for the Bottlerocket API.

Reviewer: ```suggestion
  It also contains the bootloader, the dm-verity hash tree for verifying the [immutable root filesystem](SECURITY_FEATURES.md#immutable-rootfs-backed-by-dm-verity), and the data store for the Bottlerocket API.
```

> * The data device, `/dex/xvdb`, is used for persistent storage for container orchestration, [host-containers](#Custom-host-containers), and [bootstrap containers](#Bootstrap-containers-settings).
>   It is also where container images are stored.

Reviewer: ```suggestion
* The data device, `/dex/xvdb`, is used as persistent storage for container images, container orchestration, [host-containers](#Custom-host-containers), and [bootstrap containers](#Bootstrap-containers-settings).
```

The server and client are the user-facing components of the API system, but there are a number of other components that work together to make sure your settings are applied, and that they survive upgrades of Bottlerocket.

For more details, see the [API system documentation](sources/api/).

### Default Volumes

Bottlerocket operates with two default storage volumes. 

> * The root device, `/dev/xvda`, is immutable and is mostly used for holding the OS images with its two partitions. It also contains the bootloader and the dm-verity hash tree for verifying the root filesystem image.

Reviewer 2: nits:
* one sentence per line
* there are two partition sets that each consist of multiple partitions, rather than two partitions
* settings store is also on the root device

```suggestion
* The root device, `/dev/xvda`, is immutable and is mostly used for holding the OS images with its active and passive partition sets.
  It also contains the bootloader, the dm-verity hash tree for verifying the root filesystem image, and the data store for the Bottlerocket API.
```

> * The data device, `/dex/xvdb`, is used for persistent storage for container orchestration, [host-containers](#Custom-host-containers), [bootstrap containers](#Bootstrap-containers-settings). It is also where container images are stored.

Reviewer 2: ```suggestion
* The data device, `/dex/xvdb`, is used for persistent storage for container orchestration, [host-containers](#Custom-host-containers), and [bootstrap containers](#Bootstrap-containers-settings).
  It is also where container images are stored.
```
